pom.xml

```diff
@@ -153,6 +154,11 @@
           <artifactId>apache-rat-plugin</artifactId>
           <version>${apache-rat-plugin.version}</version>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-resources-plugin</artifactId>
+          <version>${maven-resources-plugin.version}</version>
+        </plugin>
```

Comment:
I don't understand this addition: why was the `maven-resources-plugin` added? Was it to be able to specify a recent version number?

Reply:
The CI failed at "maven4" step, see last comment on [AVRO-3701](https://issues.apache.org/jira/browse/AVRO-3701), i put this plugin from 1.7.0 to 3.0.0.

Reply:
https://github.com/apache/avro/pull/2075 - I've extracted just this change, so that all other PRs won't break because of it.

Reply:
It still fails so I created https://issues.apache.org/jira/browse/MRRESOURCES-124